### ISSUE TITLE
Momentum formulation

### ISSUE THREAD

**Issue by sai-sai-sai** [mbadry1/DeepLearning.ai-Summary#171]
Say in iteration t : Vdw = Beta*Vdw + (1-Beta)*dw ---> The first term in this should actually be Vdw exponentially averaged till t-1 iterations. Since we are cacluating Vdw we can not use Vdw but need to use Vdw averaged till t-1th iteration.

**Comment by mbadry1** [comment]
Do you mean we need to write it like this:
Vdw<sub>t</sub> = Beta*Vdw<sub>t-1</sub> + (1-Beta)*dw

**Comment by sai-sai-sai** [comment]
Yes, that's what i think. What do you say?  And i think Andrew Ng also states it as you had written in his video but it may be a typo or a quick mistake overlooked.
